### PR TITLE
chore: automated PR to update ReleaseNotes for 202402.07.0 VHD

### PR DESCRIPTION
**Pull request by anujmaheshwari1**
This is an automated PR to bump ReleaseNotes for the VHD release with image version 202402.07.0

### CODE REVIEW

File: vhdbuilder/release-notes/AKSUbuntu/gen1/1804containerd/latest.txt [hunk 1]
```diff
@@ -1,9 +1,9 @@
-Starting build on  Mon Jan 22 20:42:51 UTC 2024
+Starting build on  Wed Feb 7 22:57:01 UTC 2024
 Components downloaded in this VHD build (some of the below components might get deleted during cluster provisioning if they are not needed):
   - containerd-wasm-shims v0.3.0 v0.5.1 v0.8.0
   - [installed] containerd v1.7.1-1
-  - crictl version 1.28.0
-  - runc version 1.1.7
+  - crictl version 1.29.0
+  - runc version 1.1.12-1
```
Comment (anujmaheshwari1) [inline vhdbuilder/release-notes/AKSUbuntu/gen1/1804containerd/latest.txt:6]: @cameronmeissner looking good

**Review by lilypan26 — APPROVED** [review]

File: vhdbuilder/release-notes/AKSUbuntu/gen2/2204containerd/latest.txt [hunk 2]
```diff
@@ -1,9 +1,9 @@
-Starting build on  Mon Jan 22 20:37:06 UTC 2024
+Starting build on  Wed Feb 7 22:57:23 UTC 2024
 Components downloaded in this VHD build (some of the below components might get deleted during cluster provisioning if they are not needed):
   - containerd-wasm-shims v0.3.0 v0.5.1 v0.8.0
-  - [installed] containerd v1.7.5-1
-  - crictl version 1.28.0
-  - runc version 1.1.9-2
+  - [installed] containerd v1.7.7-1
```
Comment (anujmaheshwari1) [inline vhdbuilder/release-notes/AKSUbuntu/gen2/2204containerd/latest.txt:4]: @cameronmeissner looking good for 22.04 as well

File: vhdbuilder/release-notes/AKSUbuntu/gen2/2004cvmcontainerd/latest.txt [hunk 3]
```diff
@@ -1,9 +1,9 @@
-Starting build on  Mon Jan 22 20:37:32 UTC 2024
+Starting build on  Wed Feb 7 22:56:55 UTC 2024
 Components downloaded in this VHD build (some of the below components might get deleted during cluster provisioning if they are not needed):
   - containerd-wasm-shims v0.3.0 v0.5.1 v0.8.0
-  - [installed] containerd v1.7.5-1
-  - crictl version 1.28.0
-  - runc version 1.1.9-2
+  - [installed] containerd v1.7.7-1
```
Comment (anujmaheshwari1) [inline vhdbuilder/release-notes/AKSUbuntu/gen2/2004cvmcontainerd/latest.txt:4]: @cameronmeissner correct for 20.04

File: vhdbuilder/release-notes/AKSAzureLinux/gen1/latest.txt [hunk 4]
```diff
@@ -1,11 +1,11 @@
...
 Components downloaded in this VHD build (some of the below components might get deleted during cluster provisioning if they are not needed):
   - containerd-wasm-shims v0.3.0 v0.5.1 v0.8.0
-  - [installed] containerd v1.7.5-1
-  - crictl version 1.28.0
+  - [installed] containerd v1.7.7-1
+  - crictl version 1.29.0
   - runc version 1.1.9
   - bpftrace v0.16.0
-  - nvidia-driver=-sha-e8873b
+  - nvidia-driver=-
```
Comment (cameronmeissner) [inline vhdbuilder/release-notes/AKSAzureLinux/gen1/latest.txt:8]: uhhh, this seems problematic lol

Comment (anujmaheshwari1) [reply vhdbuilder/release-notes/AKSAzureLinux/gen1/latest.txt:8]: ohh wait what

**Review by cameronmeissner — APPROVED** [review]

**Pull request merged**